commons-numbers-gamma/src/main/java/org/apache/commons/numbers/gamma/RegularizedGamma.java

```diff
@@ -190,13 +192,13 @@ public static double value(final double a,
                         /** {@inheritDoc} */
                         @Override
                         protected double getA(int n, double x) {
-                            return ((2 * n) + 1) - a + x;
+                            return n * (a - n);
                         }
 
                         /** {@inheritDoc} */
                         @Override
                         protected double getB(int n, double x) {
-                            return n * (a - n);
+                            return ((2 * n) + 1) - a + x;
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The result of `Integer` multiplication is cast into a larger type. If you think the result of the multiplication could exceed the size of an `Integer`, consider casting the arguments of the multiplication to avoid arithmetic overflows. For more information, see [CWE-190](https://cwe.mitre.org/data/definitions/190.html).